proto/rill/runtime/v1/catalog.proto

```diff
@@ -86,4 +86,8 @@ message MetricsView {
   repeated Dimension dimensions = 5;
   // Measures in the metrics view
   repeated Measure measures = 6;
+  // User friendly name
+  string display_name = 7;
```

Comment:
Should this be "label" for consistency with the terminology in the measures and dimensions?